policyengine_canada/parameters/gov/provinces/sk/tax/income/credits/sk_senior_supplementary/amount.yaml

```diff
@@ -0,0 +1,9 @@
+description:  Saskatchewan provides this amount under the senior supplement credit.
+values:
+  2023-01-01: 1_421
+metadata:
+  unit: currency-CAD
+  label: Saskatchewan Senior Supplementary tax credits return amount
+  reference:
+  - title: Saskatchewan - Senior Supplementary Tax Credits Return
```

Comment:
can you also include [this source ](https://pubsaskdev.blob.core.windows.net/pubsask-prod/806/I2-01.pdf) - we need to use it for all the PR's